topics/go/algorithms/sorting/selection/selection.go

```diff
@@ -0,0 +1,24 @@
+// Package selectionsort implement of Selection Sort algorithm in Go.
+package selectionsort
+
+// selectionSortIterative takes a random list of numbers and uses the
+// `iterative` process to sort it and return the sorted list.
+func selectionSortIterative(randomList []int) []int {
+
+	// Loop until the random list will be sorted.
+	for i := range randomList {
+		index := i
+
+		// Continue the loop until to found the smallest number in the list.
+		for j := i; j < len(randomList); j++ {
+			if randomList[j] < randomList[i] {
+				index = j
+			}
+		}
+
+		// We will swap values based on indexes.
```

Comment:
Don't say we, State what and more important why.

Reply:
I got it. I should focus more on the sentences to tell the reader what is for and why we need it.
Do you think this description is clear about what and why?
```
// Swap values based on indexes, to make the list sorted
// from the smallest number to the biggest number.
```